crates/sequencer/src/serde/utils.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::Result;
+use blockifier::execution::contract_class::{ContractClass, ContractClassV0};
+use cairo_lang_starknet::casm_contract_class::CasmContractClass;
+use starknet::core::types::{FieldElement, FlattenedSierraClass};
+
+pub fn rpc_to_inner_class(
+    contract_class: &FlattenedSierraClass,
+) -> Result<(FieldElement, ContractClass)> {
+    let class_hash = contract_class.class_hash();
+    let contract_class = rpc_to_cairo_contract_class(contract_class)?;
+    let casm_contract = CasmContractClass::from_contract_class(contract_class, true)?;
+    Ok((class_hash, ContractClass::V1(casm_contract.try_into()?)))
+}
+
+/// Converts `starknet-rs` RPC [FlattenedSierraClass] type to Cairo's
+/// [ContractClass](cairo_lang_starknet::contract_class::ContractClass) type.
+pub fn rpc_to_cairo_contract_class(
```

Comment:
why do we need this?

Reply:
required in the serialization of contract class, required from a dojo borrowed file for contract serialization.

<img width="949" alt="image" src="https://github.com/kkrt-labs/ef-tests/assets/41180869/19f72e88-a316-4c7a-96ae-c5e1f1550ec6">